src/platform/site-wide/sass/style-consolidated.scss

```diff
@@ -1,9 +1,29 @@
+//===================================
+// asset paths
+//===================================
 $font-path: "~uswds/src/fonts";
 $image-path: "~uswds/src/img";
+
+//===================================
+// color values
+//===================================
 $teamsite-dark-blue: #112e51;
 $teamsite-light-blue: #0073b4;
+
+//===================================
+// width values
+//===================================
 $teamsite-width: 959px;
 
+//===================================
+// Z-index values
+//===================================
+$base-layer: 0;
+$low-layer: 100;
+$middle-layer: 200;
+$top-layer: 300;
+$modal-layer: 400;
+
```

Comment:
I like this approach

Reply:
@RobAWilkinson came up with this for formation.